langium/er-dsl/src/language/MetaModel/Entity.ts

```diff
@@ -62,6 +72,32 @@ export class Entity {
         // return this.simpleString();
     }
 
+    public toPlantUML(): string {
```

Comment:
I suspect that this method should be renamed. It seems to me that it does not convert this entity, to puml, but rather prints the children to puml.